data/khairalanam.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "Khair Alanam",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Creative Web developer, UIUX Designer, Exploring tech in general, plays Minecraft",
+  "avatar": "https://github.com/khairalanam.png",
```

the following sections are no longer needed:
```suggestion
  "bio": "Creative Web developer, UIUX Designer, Exploring tech in general, plays Minecraft",
```

---

Okay noted.